backend/src/modules/lessons/controller.js

```diff
@@ -0,0 +1,68 @@
+import lessons from '../../models/lessons'
+
+export default class lessonsController {
+  static async create(data) {
+    const {
+      classID,
+      date,
+      topic,
+      homeWork,
+    } = data
+
+    const isTaken = await lessons.findAll({
+      where: classID, date,
```

Comment:
This will never work. Key 'where' must be an object, which has to contain classID and date.
example:
where: {
  classID,
  date,
}